Geek-Collectors-Network/geek-collectors-network
# Define resources as a seperate Docker stack

### Summary
- Suffix all resources (PMA, DB) with `-resource`
- Create targets to spin up resources, all commands do the same thing
    - `make res`
    - `make resource`
    - `make resources`
- Move build/process prereqs to the environment targets

### Testing
1. Run `make res` (or any of the other commands listed above)
2. Run `docker ps` and you should see PMA and MySQL
![image](https://github.com/Geek-Collectors-Network/geek-collectors-network/assets/1572440/fa4de7ac-1b90-4385-a9fa-b1cf279ec3fa)
